deploy/helm/flightctl/templates/flightctl-api-config.yaml

```diff
@@ -53,6 +53,7 @@ data:
         openShiftApiUrl: {{ include "flightctl.getOpenShiftAPIUrl" . }}
         internalOpenShiftApiUrl: {{ .Values.global.auth.internalOpenShiftApiUrl }}
         insecureSkipTlsVerify: {{ .Values.global.auth.insecureSkipTlsVerify }}
+        k8sRbacNs: {{ default .Release.Namespace .Values.global.auth.k8sRbacNs }}
```

Comment:
I know this is `auth` section is already ugly. Lets address it in some followup.